docs/UserGuide.md

```diff
@@ -58,105 +102,154 @@ management tasks done faster than traditional GUI apps.
-* Location matching uses each word in the String to do 'AND' search e.g. 'Room' will match 'Living room' but 'Living
-  room 1' will not match 'Living room'
-
-### Editing an item's details : `edit`
+Format: `edit INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [e/EXPIRY_DATE] [t/TAG]…​`
 
-Editing an existing item in the inventory.
+<div markdown="block" class="alert alert-info">
 
-Format: `edit INDEX [n/ITEM_NAME] [l/LOCATION] [q/QUANTITY] [e/EXPIRY_DATE] [t/TAG]…​`
+**:information_source: Notes about the edit command:**<br>
 
 * Edits the item at the specified `INDEX`. The index refers to the index number shown in the displayed item list. The
   index **must be a positive integer** 1, 2, 3, …​
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values **only if input values differ from the existing values**.
 * When editing tags, the existing tags of the item will be removed i.e. adding of tags is not cumulative.
 * You can remove all the items’ tags by typing `t/` without specifying any tags after it.
+</div>
 
 Examples:
-
-* `edit 1 q/10 e/2020-10-11` Edits the quantity of the 1st item to be `10` and expiry date of the 1st item to
-  be `2020-10-11`
-  if the existing quantity and expiry date of the 1st item are both not `10` and `2020-10-11` respectively.
-* `edit 2 n/Chocolate Bread t/` Edits the name of the 2nd item to be `Chocolate Bread` and clears all existing tags if
+* `edit 1 q/10 e/2020-10-11` edits the quantity of the 1st item to be `10` and expiry date of the 1st item to
+  be `2020-10-11` if the existing quantity and expiry date of the 1st item are both not `10` and `2020-10-11` respectively.
+* `edit 2 n/Chocolate Bread t/` edits the name of the 2nd item to be `Chocolate Bread` and clears all existing tags if
   there are existing tags and/or existing name of the 2nd item is not `Chocolate Bread`.
 
-### Locating items by name: `find`
+### 3.4. Deleting an item : `delete`
 
-Finds items whose names contain any of the given keywords, either in full or partially.
+Looking to get rid of an existing item from the inventory? Use this command.
 
-* The search is case-insensitive. e.g. `milk` will match `Milk`
-* The order of the keywords does not matter. e.g. `Bread Chocolate` will match `Chocolate Bread`
-* Only the item name is searched.
-* Items matching at least one keyword will be returned (i.e. `OR` search). e.g. `Chocolate Milk` will
-  return `Cadbury Chocolate`, `Almond Milk`
+Format: `delete INDEX`
 
-Format 1: `find KEYWORD [MORE_KEYWORDS]`
+<div markdown="block" class="alert alert-info">
 
-* Only full words will be matched e.g. `Chocolate` will not match `Chocolates`
+**:information_source: Notes about the delete command:**<br>
 
-Format 2: `find */KEYWORD [MORE_KEYWORDS]`
+* Deletes the item at the specified `INDEX`.
+* The index refers to the index number shown in the currently displayed item list.
+* The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Same reasoning on this just like the previous comment. :)

Reply:
As mentioned above, I think it is fine to leave it as it is. Perhaps, we can get the opinion of others as well.

Reply:
I think both are fine. Since it's just an additional word, I think it's ok to leave it as it is!